Alignment/CommonAlignmentProducer/plugins/GlobalTrackerMuonAlignment.cc

```diff
@@ -108,108 +114,101 @@ using namespace reco;
 
 class GlobalTrackerMuonAlignment : public edm::one::EDAnalyzer<> {
 public:
-  explicit GlobalTrackerMuonAlignment(const edm::ParameterSet&);
-  ~GlobalTrackerMuonAlignment() override = default;
-  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
-
-  void analyzeTrackTrack(const edm::Event&, const edm::EventSetup&);
-  void analyzeTrackTrajectory(const edm::Event&, const edm::EventSetup&);
+  explicit GlobalTrackerMuonAlignment(const edm::ParameterSet &);
+  ~GlobalTrackerMuonAlignment() override;
```

Comment:
what's the point of this change?

Reply:
I see this was still changed to `override`